src/main/java/cc/oakk/lotto/model/Lotto.java

```diff
@@ -0,0 +1,86 @@
+package cc.oakk.lotto.model;
+
+import cc.oakk.lotto.view.printer.Printable;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static cc.oakk.lotto.util.ValidationAdapters.throwIfNull;
+
+public class Lotto implements Printable<String> {
+    public static final int PRICE = 1000;
+    public static final int NUMBER_COUNT = 6;
+    public static final int MIN_NUMBER = 1;
+    public static final int MAX_NUMBER = 45;
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        throwIfNull(numbers);
+        if (numbers.size() != NUMBER_COUNT) {
+            throw new IllegalArgumentException(String.format("Lotto's number size must be %d!", NUMBER_COUNT));
+        }
+
+        List<Integer> validatedList = numbers.stream()
+                .map(Lotto::throwIfNotValidNumber)
+                .sorted(Lotto::compareAndThrowIfEquals)
+                .collect(Collectors.toList());
+
+        this.numbers = Collections.unmodifiableList(validatedList);
+    }
+
+    public static Lotto of(int first, int second, int third, int fourth, int fifth, int sixth) {
+        List<Integer> numbers = Stream.of(first, second, third, fourth, fifth, sixth)
+                .collect(Collectors.toList());
+        return new Lotto(numbers);
+    }
```

Comment:
만약 로또가 10개,20개 숫자를 고른다면 어떻게 될까요?
인자가 너무 많은 것 같습니다.
이런식으로 입력받고 싶으시다면, 가변인수 사용을 고려해보세요! 
```suggestion
    public static Lotto of(int... lottoNumbers) {
        List<Integer> numbers = Arrays.stream(lottoNumbers)
                .boxed()
                .collect(Collectors.toList());
        return new Lotto(numbers);
    }
```
하지만 가변인수는 주의해서 사용할 점이 있기 때문에, 배열이나 리스트로 인자를 제한하는게 좋아보이기도 합니다 :)
아래 블로그를 참고해보세요!
https://sejoung.github.io/2019/02/2019-02-19-Use_varargs_judiciously/#%EC%95%84%EC%9D%B4%ED%85%9C-53-%EA%B0%80%EB%B3%80%EC%9D%B8%EC%88%98-varargs-%EB%8A%94-%EC%8B%A0%EC%A4%91%ED%9E%88-%EC%82%AC%EC%9A%A9%ED%95%98%EB%9D%BC